src/ontology/mondo-edit.obo

```diff
@@ -436589,18 +436596,8 @@ is_obsolete: true
 [Term]
 id: MONDO:0018580
 name: obsolete PURA-related severe neonatal hypotonia-seizures-encephalopathy syndrome
-subset: ordo_disorder {source="Orphanet:438213"}
-subset: otar {source="MONDO:OTAR"}
-xref: ICD10CM:G40.4 {source="Orphanet:438213/attributed", source="Orphanet:438213/ntbt", source="Orphanet:438213"}
-xref: Orphanet:438213 {source="MONDO:obsoleteEquivalent"}
-relationship: curated_content_resource https://www.malacards.org/card/pura_related_neurodevelopmental_disorders {source="MONDO:MalaCards"}
-property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeGrouping"}
-property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/5114" xsd:anyURI
-property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6695" xsd:anyURI
-property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6752" xsd:anyURI
-property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6876" xsd:anyURI
-property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6878" xsd:anyURI
```

Comment:
PLease keep these issue links (as they are information relevant to the term). Also, please add the issue you are working on: https://github.com/monarch-initiative/mondo/issues/8953

(remember, you can add these in the mondo-edit.obo directly instead of adding them back manually)